src/Logger.php

```diff
@@ -265,19 +267,36 @@ public function getLastLogLine()
      */
     private function formatMessage($level, $message, $context)
     {
-        $level = strtoupper($level);
-        if (! empty($context)) {
+        if ($this->options['logFormat']) {
+            $parts = [

```

Comment:
Nice addition - one nit: let's use the older array syntax here since KLogger is more or less 5.3.x compatible up until this